todoapp/src/components/Header.vue

```diff
@@ -27,28 +30,38 @@ function handleAdd() {
   flex-direction: row;
   justify-content: space-between;
   align-items: center;
-  margin-bottom: 65px;
+  margin-bottom: 49px;
 }
 .header-title {
-  width: 275px;
-  height: 86px;
-  font-family: 'Neue Haas Grotesk Display Pro';
-  font-style: normal;
-  font-weight: 600;
-  font-size: 72px;
-  line-height: 86px;
+  width: 92px;
+  height: 26px;
+  font-size: 24px;
+  line-height: 29px;
   color: #000000;
 }
-
 .add-todo-btn {
   border: 0;
   display: flex;
   justify-content: center;
   align-items: center;
   cursor: pointer;
-  width: 59.64px;
-  height: 59.64px;
+  width: 32px;
+  height: 32px;
   border-radius: 50%;
   background-color: var(--add-todo-btn);
 }
+
+@media screen and (min-width: 650px) {
```

Comment:
please use a more [common breakpoint ](https://testsigma.com/blog/css-breakpoints/#:~:text=While%20there%20is%20no%20hard,1920px%20(for%20desktop%20computers).)